src/features/Tiles/PeopleTiles/styled.js

```diff
@@ -1,7 +1,6 @@
 import styled from "styled-components";
 
 export const SectionWrapper = styled.section`
-    padding: 0 16px;
```

Comment:
Now we don't need this component any more so maybe delete it and write plain `<section>` in index.js

Reply:
ok - changed for random section